neo.UnitTests/protocol.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ProtocolConfiguration": {
+    "Magic": 1234567,
+    "AddressVersion": 23,
+    "SecondsPerBlock": 2,
+    "LowPriorityThreshold": 0.001,
+    "StandbyValidators": [
+      "03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c",
+      "02df48f60e8f3e01c48ff40b9b7f1310d7a8b2a193188befe1c2e3df740e895093",
+      "03b8d9d5771d8f513aa0869b9cc8d50986403b78c6da36890638c3d46a5adce04a",
+      "02ca0e27697b9c248f6f16e085fd0061e26f44da85b58ee835c110caa5ec3ba554",
+      "024c7b7fb6c310fccf1ba33b082519d82964ea93868d676662d4a59ad548df0e7d",
+      "02aaec38470f6aad0042c6e877cfd8087d2676b0f516fddd362801b9bd3936399e",
+      "02486fd15702c4490a26703112a5cc1d0923fd697a33406bd5a1c00e0013b09a70"
+    ],
+    "SeedList": [
+      "127.0.0.1:20333",
+      "127.0.0.1:20334",
+      "127.0.0.1:20335",
+      "127.0.0.1:20336",
+      "127.0.0.1:20337",
```

Comment:
Why `127.0.0.1`?

Reply:
I just put localhost, to make sure it won't try to connect to any external ip during tests. Any suggestions?

Reply:
Does this test need to start additional nodes?

Reply:
Now that `protocol.json` is not required, we can remove most of the options, leaving only `SecondsPerBlock`.

Reply:
Thanks a lot for that Erik.